pom.xml

```diff
@@ -410,6 +416,7 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.13.0</version>
                 <configuration>
+
```

Comment:
Very small one, but I think this is a leftover of when you were trying the solution I suggested.